src/screens/SavedEventListScreen/component.js

```diff
@@ -23,11 +23,10 @@ export type Props = {
   updateData: () => Promise<void>
 };
 
-type AllProps = Props & NavigationProps;
-
-class SavedEventListScreen extends Component<AllProps> {
-  shouldComponentUpdate(nextProps: AllProps) {
-    return nextProps.isFocused;
+class SavedEventListScreen extends Component<Props> {
+  shouldComponentUpdate(nextProps: Props) {
+    // may want to omit navigation from this check
```

Comment:
It could be important to do the navigation check here. Otherwise, whenever you change save/unsave an event it will animate here.

Reply:
I think I will refactor in another PR to pull out the calls to navigate and goBack so that we can get referential equality. That would mean all pure components would work correctly.